src/Windows/Avalonia.Win32/Input/Imm32InputMethod.cs

```diff
@@ -17,7 +17,7 @@ class Imm32InputMethod : ITextInputMethodImpl
         private WindowImpl _parent;
         private ITextInputMethodClient _client;
 
-        private Imm32CaretManager _caretManager = new();
+        private Imm32CaretManager _caretManager;
```

Comment:
Structs are special here.
`default(Imm32CaretManager)` is not the same as `new Imm32CaretManager()`.
In first case ctor won't be called, but in second case it will.
I know, that Imm32CaretManager doesn't have any public ctors, but I am not sure about removing initialization anyway. Ctor might be added in the future.